src/cloudant/_common_util.py

```diff
@@ -19,12 +19,17 @@
 
 import sys
 import platform
-from collections.abc import Sequence
+import collections
 import json
 
 from ._2to3 import LONGTYPE, STRTYPE, NONETYPE, UNITYPE, iteritems_
 from .error import CloudantArgumentError, CloudantException, CloudantClientException
 
+try:
+    Sequence = collections.abc.Sequence  # noqa
```

Comment:
seems like this causes another linter issue; maybe
`sequence = collections.abc.Sequence` will be better, though of course requires some further changes.